chunky/src/java/se/llbit/chunky/ui/render/tabs/WaterTab.java

```diff
@@ -168,11 +169,18 @@ public void initialize(URL location, ResourceBundle resources) {
       }
     });
 
+    waterWorldModeDetailsPane.visibleProperty().set(waterPlaneEnabled.isSelected());
+    waterWorldModeDetailsPane.expandedProperty().set(waterPlaneEnabled.isSelected());
+    waterWorldModeDetailsPane.managedProperty().set(waterPlaneEnabled.isSelected());
+
```

Comment:
I really liked the binding more: https://github.com/chunky-dev/chunky/pull/1455/files#diff-4441e3bab76e7447488f3d3b6642671320550af35f87af8976250f0d533f75dbR177

Could we use that instead of having to call `.set` everywhere?

Reply:
The issues with using `.bind` were that attempting to collapse a panel that was expanded would fail and would cause an error in the debug console, and that using `.bind().not()` to manage the 'collapsibleProperty' would cause one of a few possible issues, one of them being that the panel contents would fail to render.